<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>YohaneWindow</class>
 <widget class="QWidget" name="YohaneWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>760</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>760</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>760</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>YohaneWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QFrame" name="mainFrame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>960</width>
     <height>720</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:qlineargradient(spread:pad,x1:0,y1:0,x2:1,y2:1,stop:0 rgba(249,241,239,1),stop:1 rgba(243,242,248,1));

border-radius:8px</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QLabel" name="dp_val">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>380</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>思源黑体</family>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);</string>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>---/---</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="dp_locker">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>380</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>思源黑体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:#00a1e8;
	color:#fff;
	border-bottom:1px solid #ddd;
	border-right:1px solid #ddd;
}

QPushButton:hover{
	background-color:#10b1f8;
}</string>
    </property>
    <property name="text">
     <string>锁定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hp_locker">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>340</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>思源黑体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:#00a1e8;
	color:#fff;
	border-bottom:1px solid #ddd;
	border-right:1px solid #ddd;
}

QPushButton:hover{
	background-color:#10b1f8;
}</string>
    </property>
    <property name="text">
     <string>锁定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="minimize">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>0</y>
      <width>45</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>思源黑体</family>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgba(0, 0, 0,0);
	border-radius:0px;
}
QPushButton:hover{
	background-color: rgba(224, 216, 213,1);
}</string>
    </property>
    <property name="text">
     <string>—</string>
    </property>
    <property name="iconSize">
     <size>
      <width>28</width>
      <height>28</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="close">
    <property name="geometry">
     <rect>
      <x>915</x>
      <y>0</y>
      <width>45</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>思源黑体</family>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgba(0, 0, 0, 0);
	border-top-left-radius:0px;
	border-bottom-left-radius:0px;
	border-bottom-right-radius:0px;
}
QPushButton:hover{
	background-color: rgba(232, 17, 35,1);
	color:#fff;
}</string>
    </property>
    <property name="text">
     <string>×</string>
    </property>
    <property name="iconSize">
     <size>
      <width>28</width>
      <height>28</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="game_status_val">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>260</y>
      <width>260</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>思源黑体</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);</string>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>未检测到游戏运行</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="game_status">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>260</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>思源黑体</family>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);</string>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>游戏状态</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="hp_label">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>340</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>思源黑体</family>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);</string>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>生命值(HP)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="dp_label">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>380</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>思源黑体</family>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);</string>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>黑暗值(DP)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="pkg_img">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>510</width>
      <height>720</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-top-right-radius:0px;
border-bottom-right-radius:0px;
background:url(:/image/assets/productPKG.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="software_info">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>695</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>思源黑体</family>
      <pointsize>9</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);</string>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>Developed by 738NGX</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="hp_val">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>340</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>思源黑体</family>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);</string>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>---/---</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="software_info_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>695</y>
      <width>300</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>思源黑体</family>
      <pointsize>9</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);</string>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>幻日夜羽 -湛海耀光- 修改器</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="header">
    <property name="geometry">
     <rect>
      <x>635</x>
      <y>90</y>
      <width>200</width>
      <height>120</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/image/assets/yhn.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="coin_label">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>440</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>思源黑体</family>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);</string>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>金币数量</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="coin_val">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>440</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>思源黑体</family>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);</string>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>￥-------</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="coin_adder">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>440</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>思源黑体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:#00a1e8;
	color:#fff;
	border-bottom:1px solid #ddd;
	border-right:1px solid #ddd;
}

QPushButton:hover{
	background-color:#10b1f8;
}</string>
    </property>
    <property name="text">
     <string>增加</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>close</sender>
   <signal>clicked()</signal>
   <receiver>YohaneWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>977</x>
     <y>14</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>minimize</sender>
   <signal>clicked()</signal>
   <receiver>YohaneWindow</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>932</x>
     <y>14</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_hp_locker_clicked()</slot>
  <slot>on_dp_locker_clicked()</slot>
 </slots>
</ui>
